test/mailers/previews/feedback_mailer_preview.rb

```diff
@@ -0,0 +1,10 @@
+# Preview all emails at http://localhost:3000/rails/mailers/feedback_mailer
+class FeedbackMailerPreview < ActionMailer::Preview
+  def feedback_email
+    msg = 'This is a message. <p>It is important</p>'
```

Comment:
Sort of. It accepts anything but it escapes when sending the email so it shouldn't render in your email client. You'll see that they submitted HTML, but it'll be like ```<p>hi!</p>```